samples/video/capture_and_display/README.rst

```diff
@@ -0,0 +1,66 @@
+.. _video_capture_display-sample:
+
+Video Capture And Display
+#########################
+
+Description
+***********
+
+This sample application uses the Video API to retrieve video frames from the
```

Comment:
This sample is not generic. It can only be used with LCDIF on IMXRTxxxx. But it could be almost generic if it would use display driver API and configure VIDEO_SW_GENERATOR to output compatible frames (example https://github.com/jfischer-phytec-iot/zephyr/blob/test-video2display/samples/video/video2display/src/main.c). Almost, because the VIDEO_SW_GENERATOR is (again) half-baked and does not support different pixel formats.

Additional:  this sample (and `samples/video/capture`) should be in **samples/drivers** because it is a sample how to use video driver API.

Reply:
> But it could be almost generic if it would use display driver API and configure VIDEO_SW_GENERATOR to output compatible frames

I'm not against such a sample that use the display API, it was just not optimized and synchronized enough for the 'zero-copy' use case.

> because the VIDEO_SW_GENERATOR is (again) half-baked and does not support different pixel formats.

What do you mean, and in which way the display API solves that issue?